public/js/index.js

```diff
@@ -147,10 +147,11 @@
     });
   });
 
-  $('form.donation-variable').on('submit', function(event) {
+  $('button.donation-variable').on('click', function(event) {
     event.preventDefault();
+    var productId = $(this).data('product-id');
```

Comment:
Oh and also apparently if you use `$foo.data()` to get all the data attrs only the camel-case keys will exist not hyphenated. See http://jsbin.com/dakibofuxa/edit?html,js,console